### DISCUSSION TITLE
Data protection - Data removal

### DISCUSSION THREAD

**Discussion by JayBeeDe** [community/community#30797] [Other Feature Feedback, Questions, & Ideas]
Hi,

I have wrongly uploaded an image that contain sensitive informations in an issue on a public repo: https://github.com/ovh/manager/issues/7866
As you can see, I have edited my message and replaced with a new screenshot.
Nevertheless the old link to the old image still exist:
`https://user-images.githubusercontent.com/xxxxxxxx/xxxxxxxx-xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxx.png`

Can you remove that file please ? Thanks

**Comment by airtower-luna** [comment]
You have now published the URL of the sensitive data to a *public* forum, with a note that it might be interesting. You might want to take appropriate action. :warning: 

For your actual request you'll have to [contact GitHub support](https://support.github.com/request) about, us other users can't help there. Good luck!

**Reply by JayBeeDe** [reply]
Is not so sensitive. But in case one day it's happening with very sensitive one, I want to know the procedure...
I have already read the [Github support page](https://support.github.com/request) This is the why I have posted a message here. I don't see any other way than posting a public message to contact Github.
Which people should I contact in such situation ?

**Reply by airtower-luna** [reply]
From the messages I've seen here a lot of people tend to get confused by that big green button that leads to the community forum. :sweat_smile: 

You need to choose the right category (and possibly sub-category) on the support page until you get to a contact form. I believe in your case it's "[remove data](https://support.github.com/request/remove-data)".
